setup.py

```diff
@@ -97,16 +97,21 @@
             "pydivert>=2.0.3,<2.2",
         ],
         'dev': [
+            "awscli>=1.19,<2",
             "hypothesis>=5.8,<6.11",
             "parver>=0.1,<2.0",
             "pdoc>=4.0.0",
+            "pip-tools>=6.1.0,<6.2",
```

Comment:
nitpick: I tend to use `>=x.y` instead of `>=x.y.z` if there is no specific `z` requirement. Might be nice to unify everything to one style. You already do so for the upper bound.